bundles/org.aposin.mergeprocessor/src/org/aposin/mergeprocessor/configuration/Configuration.java

```diff
@@ -126,7 +117,16 @@ public Configuration(final CredentialStore credentialStore) {
 
     private static final String LOCAL_H2_RENAME_DB = "rename2.mv.db";
 
-    /**
+    @Inject
+	public Configuration(final CredentialStore credentialStore) {
```

Comment:
This looks like shifted with respect to the rest of the file - could you fix the indentation?

Reply:
Fixed and squashed.